.github/workflows/push-to-prod-draft.yml

```diff
@@ -74,13 +46,13 @@ jobs:
         if: matrix.sbc == 'rockpi' && success()
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN}}
-          balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }}${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
+          balena_command: "deploy nebraltd/helium${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
       - name: Balena Deploy (Nebra RasPi)
         uses: nebraltd/balena-cli-action@v13.3.0
         if: matrix.sbc == 'raspi' && success()
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN_1}}
-          balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }} --logs --debug --nocache --build --draft"
+          balena_command: "deploy nebraltd/helium-light-hotspot --logs --debug --nocache --build --draft"
```

Comment:
Should this be `helium-light-test`? I'm currently using `helium-light-hotspot` to test [this flow](https://docs.google.com/document/d/1ZENpXrNmAcJoPIRwGVSc96tyclNTbJWZCgzc0S9vYvA/edit#) which is different.

Reply:
This was for production, not testnet.